rfcs/rfc1/README.md

```diff
@@ -0,0 +1,211 @@
+
+While the current informal approach may be more familiar and require less upfront effort, it lacks the benefits of a structured process in terms of transparency, collaboration, decision-making, and accountability. The RFC process strikes a balance between providing a clear framework for proposing and evaluating changes while still allowing for flexibility and agility in the development lifecycle.
+
+### <ins>Specify different types of RFCs</ins>
+
+As an alternative, we could adopt a categorization similar to [Python's PEP Types](https://legacy.python.org/dev/peps/pep-0001/#pep-types), which specifies three kinds of RFCs:
+
+1. A **Standards Track** RFC describes a new feature or implementation for the specifications or SDKs maintained by TBD. It may also describe an interoperability standard that will be supported outside the core libraries for current versions before a subsequent RFC adds core library support in a future version.
+
+2. An **Informational** RFC describes a design issue or provides general guidelines or information to the TBD community but does not propose a new feature.
+
+3. A **Process** RFC describes a process surrounding the development and maintenance of specifications and SDKs, or proposes a change to (or an event in) a process. Process RFCs are like Standards Track RFCs but apply to areas other than the specifications or SDKs themselves. They may propose an implementation, but not to the codebase. Examples include procedures, guidelines, and changes to the decision-making process.
+
+By adopting this categorization, the RFC process can differentiate between proposals that introduce new features or implementations (Standards Track), those that provide general information or guidelines (Informational), and those that propose changes to processes or procedures (Process). This distinction can help set expectations and guide the review and decision-making process for each type of RFC.
+
+## Prior art
+
+* [Rust RFC Process](https://github.com/rust-lang/rfcs), including [RFC-0002](https://github.com/rust-lang/rfcs/blob/master/text/0002-rfc-process.md), [RFC-1068](https://github.com/rust-lang/rfcs/blob/HEAD/text/1068-rust-governance.md)
+
+* [PEP-0001](https://legacy.python.org/dev/peps/pep-0001/), Python PEP Purpose and Guidelines
+
+* [RFC 7322](https://www.rfc-editor.org/rfc/rfc7322.html), IETF RFC Style Guide
+
+* [BIP-1](https://github.com/bitcoin/bips/blob/master/bip-0001.mediawiki) and [BIP-2](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki), Bitcoin Improvement Proposal Purpose and Guidelines
+
+* [SIP-1](https://github.com/TBD54566975/ssi-service/blob/main/doc/sip/sips/sip1/README.md), SSI Improvement Proposal Purpose and Guidelines
+
+## Unresolved questions
+
+* Does this RFC Process strike a favorable balance between formality and agility?
```

Comment:
takes everything not to insert a `u` into this blasphemous spelling of favourable